fastlane/lib/fastlane/lane_manager.rb

```diff
@@ -163,7 +163,10 @@ def self.choose_lane(ff, platform)
 
     def self.load_dot_env(env)
       # search for dotenvs in either fastlane or the current dir
```

Comment:
I think this comment should change to `# search for dotenvs in either fastlane folder or parent`